powerform/pf-deployment-context/functions/public/new-pfdeploymentcontext.ps1

```diff
@@ -9,42 +9,40 @@ Function New-PfDeploymentContext
         [string] $AzConfigFile
     ) 
     
-    $global:_PfConfiguration=[PfConfiguration]::new()
+    $global:_PfDeploymentContext=[PfDeploymentContext]::New()
```

Comment:
wonder if places reading config file need to wrap in a try to have clean message for content formatting issues

Reply:
Agreed.  Added to board